apps/are_form/models/exclusion_dates.js

```diff
@@ -82,7 +82,12 @@ module.exports = class ExclusionDates {
 
   async saveExclusionDays() {
     try {
-      const response = await axios.get(bankHolidaysApi);
+      const model = new Model();
+      const params = {
+        url: bankHolidaysApi,
+        method: 'GET'
+      };
+      const response = await model._request(params);
       const data = response.data;
 
       if (!_.get(data, `[${this.country}].events`)) {
```

Comment:
Nice work, out of scope in your work but I just noticed this about the below
```
if (!_.get(data, [${this.country}].events)) {
  return Promise.reject(new Error('Failed to retrieve data from Bank Holidays API'));
}
```
I think this will handle the error more gracefully

Reply:
👍🏾 I have updated this now